tests/requirements.txt

```diff
@@ -2,4 +2,5 @@ notebook~=7.0
 jupyterlab~=4.0
 mitosheet-helper-enterprise
 requests
-streamlit==1.31
\ No newline at end of file
+streamlit==1.31
+pytest-asyncio>=0.23.0
```

Comment:
This is the requirements.txt file for the playwright tests. It looks like `pytest-asyncio` is already included for the mito-ai project in the pyproject.toml

```
[project.optional-dependencies]
test = [
    "pytest==8.3.4",
    "pytest-asyncio==0.25.3",
    "mypy>=1.8.0",
    "types-tornado>=5.1.1",
    "types-requests>=2.25.0"
]
```